geoapps/utils/testing.py

```diff
@@ -491,7 +491,7 @@ def check_target(output: dict, target: dict, tolerance=0.1):
             / target["data_norm"],
             tolerance,
         )
-
+    print(output["phi_m"], target["phi_m"])
```

Comment:
cleanup?  Or, it seems like this could be useful - maybe we should add target information to the top print statement?

Reply:
Done